db/migrate/20190914204954_replace_assignments_and_grade_entry_forms_with_assessments.rb

```diff
@@ -30,7 +30,7 @@ def change
       t.string :short_identifier, null: false
       t.string :type, null: false
       t.string :description, null: false
-      t.text :message, null: false
+      t.text :message
```

Comment:
Keep the `null: false`, but you can have this be an empty string as a default value.